eml/api/api.js

```diff
@@ -295,26 +221,9 @@ export const getVideoStreamUrl = (fileName, resolution) => {
     resolutionPostfix = '_360x640';
   }
 
-  const _vidUrl = `${url}/api/bucket/stream/${fileName}${resolutionPostfix}.mp4`;
-  return _vidUrl;
```

Comment:
why delete this?

Reply:
Again assigning the value to a variable before returning it. Redundant step removed